IBM/carbon-addons-iot-react
# feat(tilecatalog): add error state, add blurb to wizardinline and fix padding

**Summary**

- feat(tilecatalog): add error state to tile catalog
- feat(wizardinline): add blurb to wizard inline under progress bar

**Acceptance Test (how to verify the PR)**

- Verify the new error state story of tilecatalog
- Verify blurb in Stateful Wizard story
